scripts/localbuild.sh

```diff
@@ -109,20 +125,24 @@ while getopts f:b:a:s arg; do
       ;;
     a)
       echo 'BUILDING ALL RELEASES FROM KNATIVE/DOCS'
-      # If 'true', all knative/docs branches are built to mimic a 
-      # "production" build. 
-      # REQUIRED: If you specify a fork ($FORK), all of the same branches 
+      # If 'true', all knative/docs branches are built to mimic a
+      # "production" build.
+      # REQUIRED: If you specify a fork ($FORK), all of the same branches
       # (with the same branch names) that are built in knative.dev must
-      # also exist and be available in the that $FORK (ie, 'release-0.X'). 
-      # See /config/production/params.toml for the list of the branches
-      # their names that are currently built in knative.dev.
+      # also exist and be available in the that $FORK (ie, 'release-0.X').
+      # See scripts/docs-version-settings.sh for the list of the built branches.
       BUILDALLRELEASES="${OPTARG}"
       BUILDENVIRONMENT="production"
       BUILDSINGLEBRANCH="false"
       ;;
     s)
-      echo 'Running hugo in server mode'
+      echo 'Running Hugo server'
       SERVER="server"
+      if [ "${OPTARG}" = "reload" ]; then
```

Comment:
primary change (2/3) -  if you use the "reload" flag, enable live reload in Hugo server